saltgui/static/scripts/RunType.js

```diff
@@ -43,7 +43,12 @@ export class RunType {
   }
 
   static setRunTypeDefault () {
-    RunType.menuRunType._value = "normal";
+    // Retrieve last used runType
+    RunType.menuRunType._value = Utils.getStorageItem("local", "runtype");
+    // Set default if previous runtype not stored
+    if (RunType.menuRunType._value != "normal" && RunType.menuRunType._value != "async") {
+      RunType.menuRunType._value = "normal";
+    }  
```

Comment:
that's correct, but usually any logic is done using local variables.
then when the correct value is there, assign to the intended object property
(please use with the original comments)
```
let runType = Utils.getStorageItem("local", "runtype");
if (runType !== "normal" && runType !== "async") {
    runType = "normal";
}
RunType.menuRunType._value = runType;
``` 

note also that "!==' is preferred instead of "!="
please run "npm run eslint" before committing
(and then also find some trailing spaces...)

Reply:
Makes sense. define once use many.

Fixed and linted.